[Instrumentation.AspNet] Include some attributes in sampling process (#3151)
open-telemetry/opentelemetry-dotnet-contrib: src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/CHANGELOG.md

## Unreleased

* **Breaking Change**: This module is no longer responsible for creating activities.
  The contract of `TelemetryHttpModuleOptions.OnRequestStartedCallback` was changed

>   to `Func<HttpContextBase, ActivityContext, Activity?>?`. It means that the consumer
>   is responsible for providing function returning `Activity`.

Reviewer: ```suggestion
  to `Func<HttpContextBase, ActivityContext, Activity?>?`. The consumer is now
  responsible for providing function returning `Activity`.
```
